src/views/Home.vue

```diff
@@ -1,18 +1,88 @@
 <template>
   <div class="home">
-    <img alt="Vue logo" src="../assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
+    <h1>ToDoリスト</h1>
+    <label><input type="radio" value="すべて" v-model="filter" />すべて</label>
+    <label><input type="radio" value="作業中" v-model="filter" />作業中</label>
+    <label><input type="radio" value="完了" v-model="filter" />完了</label>
+    <table>
+      <thead>
+        <tr>
+          <th>ID</th>
+          <th>コメント</th>
+          <th>状態</th>
+          <th></th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="todo in todosFilter" :key="todo.id">
+          <td>{{ todo.id }}</td>
+          <td>{{ todo.task }}</td>
+          <td>
+            <button @click="workTask(todo.id)">{{ todo.status }}</button>
+          </td>
+          <td>
+            <button @click="deleteTask(todo.id)">削除</button>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <h2>新規タスクの追加</h2>
+    <input type="text" id="task" v-model="inputTask" ref="focusThis" />
+    <button id="add" @click="addTask">追加</button>
   </div>
 </template>
 
 <script>
-// @ is an alias to /src
-import HelloWorld from '@/components/HelloWorld.vue'
+import definition from '@/helpers/definition';
 
 export default {
-  name: 'Home',
-  components: {
-    HelloWorld
-  }
-}
+  name: 'home',
+  data() {
+    return {
+      inputTask: '',
+      todos: [],
+      id: '',
+      task: '',
+      status: '',
+      deleteFlag: false,
+      filter: 'すべて',
+    };
+  },
+  methods: {
+    addTask() {
+      const obj = definition.addTask({
+        todos: this.todos,
+        task: this.inputTask,
+      });
+      this.todos = obj.todosAdd;
```

Comment:
コメントいただいたobjという命名に関してですが、こちらも具体的な命名ができると良いです。
が、少し話は逸れるのですがdefinition.addTaskの返却値をtodoの配列にしてしまうほうがスマートかなと思います。
わざわざオブジェクトで包む必要は無いかと思うので。
そうするとobjを削除でき、以下のように定義できます。
```suggestion
      this.todos = definition.addTask({
        todos: this.todos,
        task: this.inputTask,
      });
```